provision/acc_provision/acc_provision.py

```diff
@@ -487,6 +512,10 @@ def config_adjust(args, config, prov_apic, no_random):
             adj_config['cf_config']['default_endpoint_group']['app_profile'],
             adj_config['cf_config']['node_epg']))
 
+    if config["aci_config"]["vmm_domain"]["type"] == "Kubernetes":
```

Comment:
Per discussion, it seems this check should be removed.

Reply:
Removed.